style/main.css

```diff
@@ -7,34 +61,36 @@ div {
 html, body {
   margin: 0;
   padding: 0;
-  background: #faf8ef;
-  color: #776e65;
   font-family: "San Francisco", "Helvetica Neue", Arial, sans-serif;
   font-size: 15px;
-  transition: background-color 0.5s, color 0.5s;
 
   /* Android bottom padding */
   /*padding-bottom: 250px;*/
 }
-html.darkMode, body.darkMode {
-  background: #3b3b39;
-  color: #cac8bf;
+
+body {
+  background: var(--body-background-color);
+  color: var(--color);
+  transition: background-color 0.5s, color 0.5s;
+
+  /* Android bottom padding */
+  /*padding-bottom: 250px;*/
 }
 
 #mainWrapper {
   max-width: 1198px;
   min-width: 306px;
+  min-height: calc(100vh - 32px);
   margin: 32px auto 0 auto;
 }
 
 .footer {
 	clear: both;
 	padding-top: 10px;
 	margin-top: 10px;
-  margin-bottom: 40px;
 	text-align: left;
   max-width: 612px;
-  min-height: 200px;
+  min-height: 40px;
```

Comment:
This makes the replay controls overlap the bottom of the page on mobile. I moved it down to 100px for now, the replay control messages shouldn't overflow onto too many lines to cover that up. But a better way to fix that issue would be good.